src/App.tsx

```diff
@@ -1,16 +1,12 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
-import React from 'react';
-import HomePage from './components/HomePage/HomePage';
+import React, { FC } from 'react';
```

Comment:
order of imports should be:
1. global imports, e.g. React
2. local imports, './smthg'

Reply:
Fixed in commit 10558d1